circle.yml

```diff
@@ -11,19 +11,21 @@ machine:
     PROJECT_ROOT: "$GOPATH/src/github.com/moby/hyperkit"
     PATH: "${PATH}:${GOPATH}/bin"
     MACOSX_DEPLOYMENT_TARGET: "10.10"
+    OPAMVERBOSE: "1"
+    OPAMYES: "1"
 test:
   override:
     - make clean
     - make all
     - test ! -s build/lib/mirage_block_ocaml.cmi || (echo "qcow libraries have been pre-installed in CI environment" && exit 1)
     - make test
-    - brew install opam libev
-    - opam init --yes
-    - opam install --yes uri qcow.0.10.0 qcow-tool mirage-block-unix.2.7.0 ocamlfind conf-libev logs fmt mirage-unix prometheus-app
-    - make clean
-    - eval `opam config env` && make all
-    - make test
-    - eval `opam config env` && make test-qcow
+    - brew install opam
+    - opam init
+    - opam config exec -- opam depext -i uri qcow.0.10.3 conduit.1.0.0 lwt.3.1.0 qcow-tool mirage-block-unix.2.7.0 ocamlfind conf-libev logs fmt mirage-unix prometheus-app
```

Comment:
That's what I thought too. I was having strange `PATH` problems (I think) where the `opam-depext` command couldn't be found. After this is merged I'll make a PR to remove this bit and see what happens.